daemon/cmd/daemon_main.go

```diff
@@ -1452,6 +1456,15 @@ func initEnv(cmd *cobra.Command) {
 		option.Config.EnableNodePort = true
 		log.Infof("Auto-set BPF NodePort (%q) because LB IP announcements via BGP depend on it.", option.EnableNodePort)
 	}
+
+	// NetfilterCompatibleMode should be enabled only on higher versions (5.4 or later) of kernel.
+	// If kernel doesn't support higher instruction complexity limit, then disable NetfilterCompatibleMode.
+	if option.Config.NetfilterCompatibleMode {
+		if !probes.NewProbeManager().GetMisc().HaveLargeInsnLimit {
```

Comment:
This actually corresponds to [v5.2](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=c04c0d2b968ac45d6ef020316808ef6c82325a82), so we should update the comment and warning message.

Reply:
Done.